src/chat/MessageInput.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+import { Input, IconButton, InputAdornment } from '@material-ui/core';
+import SendIcon from '@material-ui/icons/Send';
+
+const submit = (event, callBack, text) => {
+    if (event.key === 'Enter') {
+        event.preventDefault();
+        event.stopPropagation();
+       callBack(text);
+    }
+}
```

Comment:
Once the fn is inside the `MessageInput` component you can adjust it to be just `submit(event)` and if its enter do the preventdefaults and `props.callBack(text)` (text is in state and callback is in props now so no need to use as param in the submit)